cli/src/config.rs

```diff
@@ -95,6 +94,13 @@ pub struct AuthConfig {
     pub refresh_token: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct AuthConfig {
+    pub okta: Option<OktaConfig>,
+    pub vault: Option<VaultConfig>,
```

Comment:
is it `vault` or `bunker`? because it is `[auth.bunker]` in the RFC

Reply:
Need to confirm this with @onimsha 
Yeah all of our code refers to vault, but new RFC says bunker. if we need to use new term we should update all the codes to avoid any confusion